Swift Code Challenge/ViewModels/UsersListViewModelAsync.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UsersListViewModelAsync.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 30/06/22.
+//
+import Foundation
+import SwiftUI
+
+class UsersListViewModelAsync: ObservableObject {
+    @Published var personList: [Person] = []
+    
+    init() {
+        getPersonList()
+    }
+    
+    func getPersonList() {
+        Task {
+            do {
+                let requestApi = try await ApiRequestAsync.fetchUser()
```

Comment:
Our naming should always be clear so if we need to come back later or if someone else sees it they could easily identify the objects and their use. Naming and documentation, `///` are essential for build sustainable code. 

```suggestion
                let requestApi = try await ApiRequestAsync.fetchUser()
```